kernel/libs/atomic-integer-wrapper/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "atomic-integer-wrapper"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+proc-macro = true
+
+[dependencies]
+proc-macro2 = "1.0.86"
+quote = "1.0.37"
+syn = { version = "2.0.77", features = ["full"] }
```

Comment:
Try not to include unused features. I heard that the `syn` crate is very large and has a lot of features. Using only the necessary features can speed up the compilation process. Could you try use less features and see if it speeds up the compilation process?

Reply:
Lots of our crates and **dependencies** are using `syn`, I think it's of little help to only disable some default features on our side since cargo workspace will finally compile with a minimal superset of all set features.

Reply:
Ok